src/_action.ts

```diff
@@ -20,10 +21,11 @@ export class Action {
     return { command };
   }
 
-  public static fromHttpGet(container: Container, path: string, options: { port?: number } = { }): k8s.HttpGetAction {
+  public static fromHttpGet(container: Container, path: string, options: { port?: number; scheme?: HttpGetScheme } = { }): k8s.HttpGetAction {
     return {
       path,
       port: k8s.IntOrString.fromNumber(options.port ?? container.port ?? 80),
+      scheme: options.scheme,
```

Comment:
We try surfacing k8s defaults as much as possible. It makes the manifest more clear, without having to wonder "what is the k8s default for this property". 

```suggestion
      scheme: options.scheme ?? HttpGetScheme.HTTP,
```